src/components/SpaceSidebarNavigation.vue

```diff
@@ -22,10 +22,41 @@ const isAdmin = computed(() => {
     admins?.includes(web3Account.value.toLowerCase())
   );
 });
+
+const mainSpace = computed(() => {
+  return props.space?.parent?.children?.find(
+    child => child.id === props.space?.id
+  )
+    ? props.space?.parent
+    : null;
+});
+
+const subSpaces = computed(() => {
+  return props.space?.children?.filter(
+    space => space.parent?.id === props.space?.id
+  );
+});
 </script>
 
 <template>
   <div>
+    <div v-if="mainSpace" class="mb-3">
```

Comment:
I'd refactor this sub-space logic into a separate component `SpaceSidebarSubspaces.vue`

Reply:
Yea I wanted to do that... but... but... well... :D